doap_HUDI.rdf

```diff
@@ -60,7 +60,7 @@
     </repository>
     <maintainer>
       <foaf:Person>
-        <foaf:name>Apache Hudi-incubating PPMC</foaf:name>
+        <foaf:name>Apache Hudi PMC</foaf:name>
```

Comment:
Line 29, the reference to incubator. is that needed?

Reply:
yes, I think we could remove it.